src/main/g8/project/Projects.scala

```diff
@@ -25,7 +25,7 @@ object Projects extends Build {
     .settings(Testing.settings:_*)
 
   lazy val examples = project.in(file("example"))
-    .dependsOn(core % "compile->compile;test->test")
+    .dependsOn(core)
```

Comment:
afaik, compile -> compile is default , I am not sure if we need the test->test one.